docs/workshop-schedule.md

```diff
@@ -24,7 +24,7 @@ All workshop sessions will take place over Zoom.
 | 3:15 PM         | Resolving merge conflicts                              | Stephanie    |
 | 3:45 PM         | _Break_                                                |              |
 | 4:00 PM         | `git` workflows and when to use them                   | Jaclyn       |
-|                 | Slides:                                                |              |
+|                 | Slides: [Git workflows](../slides/04 - Git Workflows.pdf)                                                |              |
```

Comment:
Huh! I did not know this would work with files with spaces, but it works! 

That said, I'm gonna be nitpicky and say let's pick one and use throughout - `git` vs Git`? I have no strong preference, do you?

Reply:
We're not referring to the command here, so I'd lean towards Git. I am also going to draw the line at enforcing using one throughout the slides since that doesn't strike me as a prudent use of time.

Reply:
When I start to put in my slides shortly I'll do a super quick one-over for wording!